pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -548,6 +548,14 @@ type ComponentSpec struct {
 	// - SLOW_LOG_FILE
 	// +optional
 	Env []corev1.EnvVar `json:"env,omitempty"`
+
+	// Additional containers of the component.
+	// +optional
+	Containers []corev1.Container `json:"containers,omitempty"`
+
+	// Additional volumes of the component.
+	// +optional
+	Volumes []corev1.Volume `json:"volumes,omitempty"`
```

Comment:
ditto,  if `Volumes` here is only used for sidecars, we should declare it in name.

Reply:
`additionalVolumes`?

Reply:
additionalVolumes is ok to me.